core/math/vector2.h

```diff
@@ -181,6 +159,31 @@ struct _NO_DISCARD_ Vector2 {
 	}
 };
 
+_FORCE_INLINE_ real_t &Vector2::operator[](int p_idx) {
+	DEV_ASSERT((unsigned int)p_idx < 2);
+	return coord[p_idx];
+}
+_FORCE_INLINE_ const real_t &Vector2::operator[](int p_idx) const {
+	DEV_ASSERT((unsigned int)p_idx < 2);
+	return coord[p_idx];
+}
+
+_FORCE_INLINE_ Vector2::Axis Vector2::min_axis_index() const {
+	return x < y ? Vector2::AXIS_X : Vector2::AXIS_Y;
+}
+
+_FORCE_INLINE_ Vector2::Axis Vector2::max_axis_index() const {
+	return x < y ? Vector2::AXIS_Y : Vector2::AXIS_X;
+}
+
+_FORCE_INLINE_ Vector2 Vector2::min(const Vector2 &p_vector2) const {
+	return Vector2(MIN(x, p_vector2.x), MIN(y, p_vector2.y));
+}
+
+_FORCE_INLINE_ Vector2 Vector2::max(const Vector2 &p_vector2) const {
+	return Vector2(MAX(x, p_vector2.x), MAX(y, p_vector2.y));
+}
```

Comment:
I'd restore these to their original place, less cluttered IMO, it also matches the other classes which matters